bin/index.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+const {readFile, writeFile, access, constants} = require('fs');
+const {createInterface} = require('readline');
+const {spawn} = require('child_process');
+const {promisify} = require('util');
+
+const accessAsync = promisify(access);
+const readFileAsync = promisify(readFile);
+const writeFileAsync = promisify(writeFile);
+
+const packageJSONPath = './package.json';
+
+const packages = [
+  '@autovance/eslint-config-autovance',
+  '@typescript-eslint/eslint-plugin@3',
+  'eslint@7',
+  'eslint-config-prettier@6',
+  'eslint-plugin-node@11',
+  'eslint-plugin-prettier@3',
+  'prettier@2'
+];
+
+const styleExtendType = {
+  hybrid: '@autovance/eslint-config-autovance',
+  javascript: '@autovance/eslint-config-autovance/javascript',
+  typescript: '@autovance/eslint-config-autovance/typescript'
+};
+
+async function hasValidPackageJSON(path) {
+  try {
+    await accessAsync(path, constants.W_OK);
+    return true;
+  } catch (err) {
+    return false;
+  }
+}
+
+async function prompt(question) {
+  const readline = createInterface({
+    input: process.stdin,
+    output: process.stdout
+  });
+
+  const response = await new Promise((resolve) => {
+    readline.question(question, (answer) => {
+      resolve(answer)
+      readline.close();
+    });
+  });
+  return response.trim().toLowerCase();
```

Comment:
```suggestion
  return response.trim().toLowerCase().charAt(0);
```

Since we only ever use the first letter.